test/crypto-service/test_accountCreateTransaction.js

```diff
@@ -923,27 +947,33 @@ describe("AccountCreateTransaction", function () {
       accountId,
       stakedAccountId,
     ) {
-      // If the account was created successfully, the queried account's staked account IDs should be equal.
+      // If the account was created successfully, the queried account's staked node IDs should be equal.
```

Comment:
This function is still comparing account IDs it looks like, I don't think this comment needs to be updated.